README.md

```diff
@@ -40,40 +41,40 @@ demo_project:
 ```
 
 ## Supported features
-| Name | Supported | Iceberg |
-|------|-----------|---------|
-|Materialization: View|Yes| N/A |
-|Materialization: Table|Yes| Yes |
-|Materialization: Table with Partitions |Yes| Yes |
-|Materialization: Incremental - Append|Yes| Yes |
-|Materialization: Incremental - Append with Partitions |Yes| Yes |
-|Materialization: Incremental - Insert+Overwrite |Yes| Yes |
-|Materialization: Incremental - Insert+Overwrite with Partition |Yes| Yes |
-|Materialization: Incremental - Merge|No| No |
-|Materialization: Ephemeral|Yes| Yes |
-|Seeds|Yes| Yes |
-|Tests|Yes| Yes |
-|Snapshots|No| No |
-|Documentation|Yes| Yes |
-|Authentication: LDAP|Yes| Yes |
-|Authentication: Kerberos|Yes| No |
+| Name | Supported | Iceberg | Kudu |
+|------|-----------|---------|------|
+|Materialization: View|Yes| N/A | N/A |
+|Materialization: Table|Yes| Yes | Yes |
+|Materialization: Table with Partitions |Yes| Yes | No |
+|Materialization: Incremental - Append|Yes| Yes | Yes |
+|Materialization: Incremental - Append with Partitions |Yes| Yes | No |
+|Materialization: Incremental - Insert+Overwrite |Yes| Yes | Yes |
+|Materialization: Incremental - Insert+Overwrite with Partition |Yes| Yes | No |
+|Materialization: Incremental - Merge|No| No | No |
+|Materialization: Ephemeral|Yes| Yes | No |
+|Seeds|Yes| Yes | Yes |
+|Tests|Yes| Yes | Yes |
+|Snapshots|No| No | No |
+|Documentation|Yes| Yes | Yes |
+|Authentication: LDAP|Yes| Yes | Yes |
+|Authentication: Kerberos|Yes| No | No |
 
 ### Tests Coverage
 
 #### Functional Tests
-| Name | Base | Iceberg |
-|------|------|---------|
-|Materialization: View|Yes| N/A |
-|Materialization: Table|Yes| Yes |
-|Materialization: Table with Partitions |Yes| Yes |
-|Materialization: Incremental - Append|Yes| Yes |
-|Materialization: Incremental - Append with Partitions |Yes| Yes |
-|Materialization: Incremental - Insert+Overwrite |Yes| Yes |
-|Materialization: Incremental - Insert+Overwrite with Partition |Yes| Yes |
-|Materialization: Ephemeral|Yes| Yes |
-|Seeds|Yes| Yes |
-|Tests|Yes| Yes |
-|Snapshots|No| No |
-|Documentation| Yes | Yes |
-|Authentication: LDAP|Yes| Yes |
-|Authentication: Kerberos|No| No |
+| Name | Base | Iceberg | Kudu |
+|------|------|---------|------|
+|Materialization: View|Yes| N/A | N/A |
+|Materialization: Table|Yes| Yes | Yes |
+|Materialization: Table with Partitions |Yes| Yes | No |
+|Materialization: Incremental - Append|Yes| Yes | Yes |
+|Materialization: Incremental - Append with Partitions |Yes| Yes | No |
+|Materialization: Incremental - Insert+Overwrite |Yes| Yes | Yes |
```

Comment:
Where are the test for `Insert+Overwrite` ? Most of the test are disabled.

Reply:
I was adding tests based on Impala, looks like there is no separate `Insert+Overwrite` test for Impala too, `Insert+Overwrite` is combined with `partition_key`. Let me know if a separate test is needed, I'll add for both.

Reply:
please add the tests. If is going to take long time then please plan it as a part of separate PR. Just update the docs

Reply:
Updated the docs now and created separate JIRAs to track the missing and disabled tests.